package.json

```diff
@@ -40,19 +41,17 @@
     "@testing-library/jest-dom": "5.16.2",
     "@testing-library/react": "12.1.4",
     "@types/jest": "27.4.1",
-    "@types/ramda": "0.27.45",
     "@types/ramda": "0.28.1",
-    "@types/react": "17.0.30",
     "@types/react": "17.0.41",
     "@typescript-eslint/eslint-plugin": "5.16.0",
     "@typescript-eslint/parser": "5.16.0",
     "autoprefixer": "10.4.4",
     "babel-loader": "8.2.3",
     "dependency-cruiser": "11.4.1",
     "eslint": "8.11.0",
-    "eslint-config-next": "12.1.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-prettier": "4.0.0",
+    "eslint-plugin-react": "^7.29.4",
```

Comment:
Minor but can we keep these static? (that is, without `^` preceding them?) You can also run `nps meta.bureaucracy` to do so automatically.